src/renderers/shaders/ShaderChunk/begin_vertex.glsl

```diff
@@ -1,2 +1,28 @@
 
+#ifndef INSTANCE_TRANSFORM
+
 vec3 transformed = vec3( position );
+
+#else
+
+#ifndef INSTANCE_TRANSFORM_DEFINED
```

Comment:
I don't understand this code. What's the benefit of this `INSTANCE_TRANSFORM_DEFINED` thing?